packages/wxt/e2e/tests/init.test.ts

```diff
@@ -1,18 +1,22 @@
 import { describe, it, expect } from 'vitest';
 import { TestProject, WXT_PACKAGE_DIR } from '../utils';
-import { execaCommand } from 'execa';
+import spawn from 'nano-spawn';
 import glob from 'fast-glob';
 import { mkdir, writeJson } from 'fs-extra';
 
 describe('Init command', () => {
   it('should download and create a template', async () => {
     const project = new TestProject();
 
-    await execaCommand(`pnpm -s wxt init ${project.root} -t vue --pm npm`, {
-      env: { ...process.env, CI: 'true' },
-      stdio: 'ignore',
-      cwd: WXT_PACKAGE_DIR,
-    });
+    await spawn(
+      'pnpm',
+      ['-s', 'wxt', 'init', project.root, '-t', 'vue', '--pm', 'npm'],
+      {
+        env: { ...process.env, CI: 'true' },
```

Comment:
Seems like we don't need to spread process.env into this anymore:

> `options.env`
>
> Type: `object`
> Default: `{}`
>
> Override specific [environment variables](https://en.wikipedia.org/wiki/Environment_variable). Other environment variables are inherited from the current process ([process.env](https://nodejs.org/api/process.html#processenv)).
>
> &ndash; https://www.npmjs.com/package/nano-spawn/v/0.1.0#optionsenv

Reply:
Oh, overread that oopsies, will fix